ch_tools/monrun_checks/ch_orphaned_objects.py

```diff
@@ -0,0 +1,159 @@
+from datetime import timedelta
+from tempfile import TemporaryFile
+from typing import Optional
+
+import click
+
+from ch_tools.chadmin.internal.object_storage.obj_list_item import ObjListItem
+from ch_tools.chadmin.internal.object_storage.s3_cleanup import (
+    cleanup_s3_object_storage,
+)
+from ch_tools.chadmin.internal.object_storage.utils import (
+    DEFAULT_GUARD_INTERVAL,
+    get_orphaned_objects_query,
+    get_remote_data_paths_table,
+    get_traverse_shadow_settings,
+    traverse_object_storage,
+)
+from ch_tools.chadmin.internal.utils import execute_query
+from ch_tools.common.cli.parameters import TimeSpanParamType
+from ch_tools.common.clickhouse.config import get_clickhouse_config
+from ch_tools.common.clickhouse.config.storage_configuration import S3DiskConfiguration
+from ch_tools.common.result import CRIT, OK, WARNING, Result
+
+DRY_RUN = True
+ON_CLUSTER = True
+CLUSTER_NAME = "{cluster}"
+DIST_NAME = "object_storage"
+OBJECT_NAME_PREFIX = None
+
+
+@click.command("orphaned-objects")
+@click.option(
+    "--keep-paths",
+    "keep_paths",
+    is_flag=True,
+    help=("Do not delete collected paths of objects from object storage."),
```

Comment:
This was originally misspelled. Let's fix wording now. `Do not delete the collected paths from the local table, when the command completes.`

Reply:
thanks, done